docs/user/gcp/install_upi.md

```diff
@@ -0,0 +1,594 @@
+EOF
+```
+- `infra_id`: the infra_id from above
+- `region`: the region to deploy the cluster into (for example us-east1)
+- `cluster_network`: the URI to the cluster network
+- `network_cidr`: the CIDR of the vpc network (for example 10.0.0.0/16)
+- `master_nat_ip`: the ip address of the master nat (for example 34.94.100.1)
+- `worker_nat_ip`: the ip address of the worker nat (for example 34.94.200.1)
+
+Create the deployment using gcloud.
+
+```sh
+gcloud deployment-manager deployments create ${INFRA_ID}-security --config 03_security.yaml
+```
+
+The templates do not create the policy bindings due to limitations of Deployment
+Manager, so we must create them manually.
+
+```sh
+export MASTER_SA=${INFRA_ID}-m@${PROJECT_NAME}.iam.gserviceaccount.com
+gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${MASTER_SA}" --role "roles/compute.instanceAdmin"
+gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${MASTER_SA}" --role "roles/compute.networkAdmin"
+gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${MASTER_SA}" --role "roles/compute.securityAdmin"
+gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${MASTER_SA}" --role "roles/iam.serviceAccountUser"
+gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${MASTER_SA}" --role "roles/storage.admin"
+
+export WORKER_SA=${INFRA_ID}-w@${PROJECT_NAME}.iam.gserviceaccount.com
+gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${WORKER_SA}" --role "roles/compute.viewer"
+gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${WORKER_SA}" --role "roles/storage.admin"
+```
+
+Create a service account key and store it locally for later use.
+
+```sh
+gcloud iam service-accounts keys create service-account-key.json --iam-account=${MASTER_SA}
+
+```
+
+## Create the cluster image.
+
+Locate the RHCOS image source and create a cluster image.
+
+```sh
+export IMAGE_SOURCE=`curl https://raw.githubusercontent.com/openshift/installer/master/data/data/rhcos.json | jq -r .gcp.url`
+gcloud compute images create "${INFRA_ID}-rhcos-image" --source-uri="${IMAGE_SOURCE}"
+```
+
+## Launch temporary bootstrap resources
+
+Copy [`04_bootstrap.py`](../../../upi/gcp/04_bootstrap.py) locally.
+
+Export variables needed by the resource definition.
+
+```sh
+export CONTROL_SUBNET=`gcloud compute networks subnets describe ${INFRA_ID}-master-subnet --format json | jq -r .selfLink`
+export CLUSTER_IMAGE=`gcloud compute images describe ${INFRA_ID}-rhcos-image --format json | jq -r .selfLink`
+```
+
+Create a bucket and upload the bootstrap.ign file.
+
+```sh
+gsutil mb gs://${INFRA_ID}-bootstrap-ignition
+gsutil cp bootstrap.ign gs://${INFRA_ID}-bootstrap-ignition/
+```
+
+Create a signed URL for the bootstrap instance to use to access the ignition
+file. Export the https URL from the output as a variable.
```

Comment:
"ignition file" -> "Ignition config".  And "https" -> "HTTPS"

Reply:
Done and removed redundant HTTPS.